guides/slido-moderation.rst

```diff
@@ -0,0 +1,102 @@
+   at the end of this guide, check with CoC or ask other organizers around you
+   or on Slack if you aren't sure), click on the green check mark to approve the
+   question (only now will it be shown publicly) or the red cross to dismiss it.
+5) After the talk ends, it's time for Q&A. Going from the top to the bottom of
+   the "Live" list, click on the double blue arrow to highlight the question
+   which the room moderator should read. After the question is answered, click
+   on the box to archive it and highlight the next one.
+6) If you accidentally dismiss a good question or archive one that wasn't
+   answered yet, go to the ``Archived`` list on the top of the right window and
+   click on the circled arrow to restore the question. Based on whether the
+   question was approved or not, it will show either in the waiting list or the
+   live stream window.
+7) If there are some unanswered questions left after Q&A ends (eg. the time ran
+   out), you can archive them by clicking on ``Archive all`` on the right side
+   of the screen (right below the room switcher), no need to archive each
+   question separately. This doesn't archive the questions in the moderation
+   queue so if there are any, you need to archive them separately. It shouldn't
+   happen to you though - try to check the questions right as they arrive so
+   people can vote on them longer!
+
+
+Question examples
+-----------------
+
+* ``The speaker put no effort into the talk and just read everything``
+
+  It doesn't matter whether the comment is true or not. Sli.do is not a way to
+  submit feedback to the talk (and even if it was it can be expressed in a nicer
+  way). This question should not be approved.
+
+* ``Are you seeing a girl?``
+
+  How is this related to the talk, and how is it appropriate to ask a speaker
+  during the conference at all? This is not Tinder and *personal questions
+  should not be approved*.
+
+* ``You have a nice beard``
+
+  Inappropriate and shouldn't be approved, no further explanation needed.
+
+* ``<speaker talks about teaching Python> How did you get to learn Python?``
+
+  While this is borderline personal question it's is still on topic for this
+  talk as it's about learning and getting people to learn; and should be
+  approved
+
+* ``<speaker talks about a technical thing> Do you know library XYZ which does
+  something similar?``
+
+  Straightforward question that's on topic that definitely should be approved.
+
+* ``Nice talk``
+
+  This is a comment that doesn't have any value and shouldn't be approved.
+
+* ``Nice talk! I have one comment, you mentioned project ABC didn't support
+  Python3 but they actually added the support a few months ago``
+
+  While it's a comment and not a question, it should be approved as it politely
+  points out something the audience should know that's related to the talk.
+  As you can see in the examples it's usually pretty clear if the question is
```

Comment:
I think this line and the rest of the others should be a start of a new paragraph or perhaps a `.. note::` block, because it comments the whole examples question, not just the last one.

Reply:
Added it as a note